packages/clay-color-picker/src/__tests__/index.tsx

```diff
@@ -393,6 +396,35 @@ describe('Interactions', () => {
 
 			expect(document.body).toMatchSnapshot();
 		});
+
+		it('pressing right arrow key increase hue value', async () => {
+			const hueSlider = screen.getByRole('slider');
+
+			userEvent.click(hueSlider);
+			expect(handleColorsChange).toBeCalledTimes(1);
+			expect(handleValueChange).toBeCalledTimes(1);
+
+			userEvent.keyboard('[ArrowRight]');
+
+			expect(handleColorsChange).toBeCalledTimes(2);
+			expect(handleValueChange).toBeCalledTimes(2);
```

Comment:
Hey @matuzalemsteles 
The reason I had to add two checks on this test was because to set focus on the slider, I had to  use a click and that triggers the `onChange` function and when i press the arrow key, it triggers again.